pumpkin-inventory/Cargo.toml

```diff
@@ -4,10 +4,9 @@ version.workspace = true
 edition.workspace = true
 
 [dependencies]
-# For items
-pumpkin-world = { path = "../pumpkin-world"}
-
 num-traits = "0.2"
 num-derive = "0.4"
-
-
+thiserror = "1.0.63"
```

Comment:
i really should make thiserror a workspace dependency. For now you should just use `thiserror = "1.0"`